data_science/specs/cohort_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rspec'
+require 'json'
+require_relative '../cohort.rb'
+
+RSpec.describe 'Cohort' do
+  let (:test_data) { JSON.parse(File.read('../test_data.json')) }
+  let (:cohort_a) { test_data.select { |v| v['cohort'] == 'A' } }
+
+  before :each do
+    @cohort = Cohort.new(name: 'A', data: cohort_a)
+  end
+
+  describe '#new' do
+    it 'returns a new Cohort object' do
+      expect(@cohort).to be_an_instance_of Cohort
+    end
+
+    it 'initializes with a string for a name' do
+      expect(Cohort.new.name).to be_a String
```

Comment:
This test is pretty meaningless. The equality test does everything this does in a much better way: if it's the wrong type, equality will break anyway and give you a message that's just as useful.